docs/src/guide/tutorial.jl

```diff
@@ -27,13 +27,22 @@
 
 using DFTK
 using Plots
+using Unitful
+using UnitfulAtomic
 
 ## 1. Define lattice and atomic positions
-a = 10.26  # Silicon lattice constant in Bohr
+a = 5.431u"angstrom" # Silicon lattice constant
 lattice = a / 2 * [[0 1 1.];  # Silicon lattice vectors
                    [1 0 1.];  # specified column by column
                    [1 1 0.]]
 
+# By default, all numbers passed as arguments are assumed to be in atomic units.
+# Quantities such as temperature, energy cutoffs, lattice vectors, and the
+# k-point grid spacing can optionally be annotated with Unitful units, which are
+# automatically converted to the atomic units used internally. For Unitful
+# package documentation, see https://painterqubits.github.io/Unitful.jl/stable/
+# and https://github.com/sostock/UnitfulAtomic.jl.
```

Comment:
Does that directly render as a clickable link in the html? I think I would still shorten it to sth like:
```
For more details on this see the [Unitful package documentation](https://painterqubits.github.io/Unitful.jl/stable/) and
the [UnitfulAtomic.jl package](https://github.com/sostock/UnitfulAtomic.jl).
```